src/goto-instrument/full_slicer.cpp

```diff
@@ -322,18 +329,38 @@ Function: implicit
 
 \*******************************************************************/
 
-static bool implicit(goto_programt::const_targett target)
+static bool implicit(const namespacet &ns, goto_programt::const_targett target)
 {
   // some variables are used during symbolic execution only
```

Comment:
I have added an explanation in https://github.com/diffblue/cbmc/issues/260. As I reported in the issue, I'm not completely sure whether this is correct or not.